python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -156,7 +156,7 @@ class NearestNeighbors(Base):
     n_neighbors: int (default = 5)
         The top K closest datapoints you want the algorithm to return.
         Currently, this value must be < 1024.
-    should_downcast : bool (default = False)
+    should_downcast : bool (default = None)
```

Comment:
This is deprecated, right? We should just document that it's deprecated instead of the rest.

Reply:
it is deprecated but the default for should_downcast was set to True. Therefore even if a user does not pass a value for it they would get the warning message which might confuse them. In addition, the warning would ask the user to use `convert_dtypes` in place of `should_downcast` but the documentation would not mention `convert_dtypes` anywhere.

Reply:
I was thinking the docstring should clearly say - "This parameter is deprecated."

Reply:
ohh, I dont believe its that says that at the moment. I will add that in the documentation.